code/obj/submachine/robotics.dm

```diff
@@ -122,9 +122,9 @@
 
 /obj/item/atmosporter
 	name = "Atmospherics Transporter"
-	desc = "Used by Atmospherics Cyborgs for convenient transport of siphons and tanks."
-	icon = 'icons/obj/items/items.dmi'
-	icon_state = "bedbin"
+	desc = "Used by Engineering Cyborgs for convenient transport of siphons and tanks."
```

Comment:
As you're touching this, let's remove the type of module entirely (hint: near future cyborg stuff might make this incorrect).

Reply:
I've removed the module reference in the description for the jumper cables as well (which wasn't accurate anymore anyway), should I do the same for the mini-chem? It's a case of it being in the same file anyway, but I don't know if that's too unrelated to add in here.